### PR TITLE
viomem: support viomem driver for win driver case

### PR DESCRIPTION
**Pull request by menli820**
Support viomem driver in win_sigverify and virtio_driver_sign_check test cases.

ID: 1984, 1985

Signed-off-by: menli <menli@redhat.com>

### CODE REVIEW

**Comment by menli820** [comment]
No available package to test, lack some necessary information, so change the patch status to draft first.

**Comment by menli820** [comment]
(1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.virtio_driver_sign_check.with_viomem.virtio_win_iso_media.q35: STARTED
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.virtio_driver_sign_check.with_viomem.virtio_win_iso_media.q35: PASS (99.23 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2024-06-06T09.48-904acfe/results.html
JOB TIME   : 100.72 s
Test result：


 (1/2) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (1/2) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1415.35 s)
 (2/2) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_sigverif.with_viomem.msft_sign_check.q35: STARTED
 (2/2) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_sigverif.with_viomem.msft_sign_check.q35: PASS (170.37 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2024-06-06T11.57-18c416c/results.html
JOB TIME   : 1587.66 s

**Comment by menli820** [comment]
@leidwang please help to review it when you free , thanks

File: qemu/tests/cfg/win_sigverif.cfg [hunk 1]
```diff
@@ -94,3 +94,14 @@
             numa_nodeid_shm0 = 0
             driver_name = viofs
             device_name = "VirtIO FS Device"
+        - with_viomem:
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
```
[the remark this answers is not in the document]
Comment (menli820) [reply qemu/tests/cfg/win_sigverif.cfg:98]: @leidwang  Thanks for the reminder, but for qemu side we support viomem from rhel9 , so I think we should still keep it.

[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/cfg/win_sigverif.cfg:98]: Under the current limitations, this case will run with some lower version of prewhql or rpm package, then this case will fail. This is not what we expect, right?
Other code LGTM, please request others to review this PR.Thanks.

**Comment by menli820** [comment]
@leidwang please help to review it again thanks

**Review by leidwang — APPROVED** [review]
LGTM.

**Comment by menli820** [comment]
@PaulYuuu please help to review this patch when you free, thanks

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**